tensorboard/webapp/testing/dom.ts

```diff
@@ -126,9 +126,16 @@ export function sendKey<T>(
     el.setSelectionRange(startingCursorIndex, startingCursorIndex);
   }
 
+  // Use to override default options on programmatic events to be closer to
+  // user-initiated events, which bubble and cancel.
+  const baseEventOptions = {
```

Comment:
Cool. Would you be able to spend a little bit of time to maybe find the TypeScript HTML typing for the event options? Since it is already checked when you pass it as the second argument to Event constructor, I guess it is okay as is, too.